mozilla-release/browser/confvars.sh

```diff
@@ -35,8 +35,8 @@ MOZ_SERVICES_COMMON=1
 MOZ_SERVICES_CRYPTO=1
 MOZ_SERVICES_HEALTHREPORT=1
 MOZ_SERVICES_METRICS=1
-MOZ_SERVICES_SYNC=1
```

Comment:
maybe we can find a better way to alter those vars. One that will not require changes in source files